File: junsuk/17주차/K번째수.js
function solution(array, commands) {
    return commands.map(command => {
        const [i, j, k] = command;
        
        const slicedArr = array.slice(i - 1, j);
        
        slicedArr.sort((a, b) => a - b);
        
        return slicedArr[k - 1];
    })
}

File: jin/21주차/[PGS]42748. K번째수_LV1/K번째수.java
import java.util.*;

class Solution {
    public int[] solution(int[] array, int[][] commands) {
        int[] answer = new int[commands.length];
        
        int cnt = 0;
        
        for(int[] command : commands){
            int i = command[0];
            int j = command[1];
            int k = command[2];
            
            int[] cutArr = new int[j - i + 1];
            
            int idx = 0;
            
            for(int m = i; m <= j; m++){
                cutArr[idx] = array[m - 1];
                idx++;
            }
            
            Arrays.sort(cutArr);
            
            answer[cnt] = cutArr[k - 1];
            cnt++;
        }
            
        return answer;
    }
}